source/docs/appendices/migration-guides.md

```diff
@@ -9,6 +9,24 @@ Our goal is to make migrations as smooth as possible. This is why we try to make
 
 ## `1.0.0-alpha.2` -> `1.0.0-beta.0`
 
+### Translations
```

Comment:
I think we should add a line here about updating the Magento 2 module (and updating the module version defined in the `.env`)